### ISSUE TITLE
AI Website

### ISSUE THREAD

**Issue by carmelo-cyber** [carmelo-cyber/Carmelo_GitHub#8]
<html>
<head>
    <title>AI Example</title>
</head>
<body>
    
    <form id="ai-form">
        <label for="input1">Input 1:</label>
        <input type="number" id="input1" placeholder="Enter a number here...">
        <label for="input2">Input 2:</label>
        <input type="number" id="input2" placeholder="Enter a number here...">
        <button type="submit">Predict</button>
    </form>

    
    <div id="ai-output"></div>

    
    <script>
        // Import necessary libraries
        const { datasets, train_test_split, RandomForestClassifier } = require('sklearn');

        // Load the iris dataset
        const iris = datasets.load_iris();

        // Split the data into training and test sets
        const X_train, X_test, y_train, y_test = train_test_split(iris.data, iris.target, test_size=0.3);

        // Train a random forest classifier on the training data
        const clf = new RandomForestClassifier(n_estimators=100);
        clf.fit(X_train, y_train);

        // Handle form submission
        const form = document.getElementById('ai-form');
        form.addEventListener('submit', e => {
            // Prevent the form from submitting
            e.preventDefault();

            // Get the user's inputs
            const input1 = document.getElementById('input1').value;
            const input2 = document.getElementById('input2').value;

            // Use the classifier to make a prediction
            const prediction = clf.predict([[input1, input2]]);

            // Add the prediction to the output display
            const output = document.getElementById('ai-output');
            output.innerHTML = `<p>Prediction: ${prediction}</p>`;
        });
    </script>
</body>
</html>
